### PR TITLE
Converts the table display field into a link for the view action.

### PR DESCRIPTION
**Pull request by lorenzo**
This makes it more user friendly as the area for clicking the details
of a row is bigger.

### CODE REVIEW

File: src/View/Helper/CrudViewHelper.php [hunk 1]
```diff
@@ -323,6 +329,16 @@ public function associations()
     }
 
     /**
+     * Returns the main table dispaly field column
+     *
+     * @return void
+     */
+    public function displayField()
```
Comment (AD7six) [inline src/View/Helper/CrudViewHelper.php]: Is this necessary/useful?

Comment (lorenzo) [inline src/View/Helper/CrudViewHelper.php]: I was following the style of the rest of the helper, I can gladly remove

Comment (AD7six) [inline src/View/Helper/CrudViewHelper.php]: It's this discrepancy which lead me to ask:

    if ($field === $this->displayField()) {
                          ^ explicit accessor
            $value = ... $data->get($this->getViewVar('primaryKey'))]);
                                           ^ no explicit accessor

Would be best to be consistent, but creating getters to wrap getViewVar seems a bit derp

Comment (lorenzo) [inline src/View/Helper/CrudViewHelper.php]: Agree, I will remove this case

**Comment by lorenzo** [comment]
@AD7six done

**Pull request merged**